src/test/unit_tests/test_public_access_block.js

```diff
@@ -46,7 +46,7 @@ async function run_on_random_bucket(s3, bucket_prefix, cb) {
     try {
         await cb(bucket);
     } finally {
-        await s3.deleteBucket({ Bucket: bucket });
+        await s3.deleteBucket({ Bucket: bucket }).catch(() => { /* nothing */ });
```

Comment:
why would delete bucket fail? shouldn't we handle such failure?

Reply:
Because sometimes the cb code might forget to delete all the objects (and their versions) from the bucket but that doesn't effect the correctness.

Failing to delete should be OK if the `cb` deems so. We can obviously write code to handle those cases but I think that would be more fragile without adding anything to the correctness of the respective tests. WDYT?